optimade/validator/validator.py

```diff
@@ -524,19 +525,16 @@ def _get_archetypal_entry(
             if data_returned < 1:
                 return (
                     None,
-                    "Endpoint {endp!r} returned no entries, cannot get archetypal entry.",
+                    "Endpoint {endp!r} returned no entries, cannot get archetypal entry or test filtering.",
                 )
 
-            response, message = self._get_endpoint(
-                f"{endp}?page_offset={random.randint(0, data_returned-1)}&response_fields={','.join(properties)}",
-                multistage=True,
+            archetypal_entry = response.json()["data"][
```

Comment:
This change removes the need for `page_offset` to be implemented by the API, and instead just chooses a random result from the first page of the API results to use for the filter tests.